Makefile

```diff
@@ -1,4 +1,4 @@
-  
+
```

Comment:
@bquenin @freignat91 @ndegory Alright wiseguys! :p Not that the top blank line really matters, but I'm totally obsessing over why it keeps getting reintroduced -- and it's not because of .editorconfig. It was definitely removed with [my commit](https://github.com/appcelerator/amp/blob/a7f6360f42d5b9dfac0c3ac7d6afe583492889b0/Makefile) to master yesterday, and reintroduced by @freignat91 in his [subsequent PR](https://github.com/appcelerator/amp/pull/617/files). Since I merged his PR and didn't notice, it's my bad, but also @freignat91 (other weirdness: I see lines 90 and 95 of his PR being introduced as updates for lines I already fixed a number of commits ago)! If a PR is stale at all (based on branching off a version of master that hasn't been updated at least the same day already), then you really should rebase before submitting the PR. Anyway, thanks @bquenin :)